addQuote.php

```diff
@@ -0,0 +1,131 @@
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    <option value="4">4</option>
+                    <option value="5">5</option>
+                    <option value="6">6</option>
+                    <option value="7">7</option>
+                    <option value="8">8</option>
+                    <option value="9">9</option>
+                    <option value="10">10</option>
+                    <option value="11">11</option>
+                    <option value="12">12</option>
+                    <option value="13">13</option>
+                    <option value="14">14</option>
+                    <option value="15">15</option>
+                    <option value="16">16</option>
+                    <option value="17">17</option>
+                    <option value="18">18</option>
+                    <option value="19">19</option>
+                    <option value="20">20</option>
+                    <option value="21">21</option>
+                    <option value="22">22</option>
+                    <option value="23">23</option>
+                    <option value="24">24</option>
+                    <option value="25">25</option>
+                    <option value="26">26</option>
+                    <option value="27">27</option>
+                    <option value="28">28</option>
+                </select>
+                <label for="rating">Hilaritometer Rating (1-10):</label>
+                <select id="rating" name="rating">
+                    <option hidden disabled selected value> -- select an option -- </option>
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    <option value="4">4</option>
+                    <option value="5">5</option>
+                    <option value="6">6</option>
+                    <option value="7">7</option>
+                    <option value="8">8</option>
+                    <option value="9">9</option>
+                    <option value="10">10</option>
+                </select>
+                <input type="submit" value="Submit">
+            </form>
+            <a href="collection.php"><button>Back to Collection</button></a>
+        </main>
+    </body>
+</html>
+
+<?php
+if (isset($_POST['quote']) && isset($_POST['charName']) && isset($_POST['epName']) && isset($_POST['epNum']) && isset($_POST['season']) && isset($_POST['rating'])) {
+    $quote = '"' . $_POST['quote'] . '"';
+    $character = $_POST['charName'];
+    $episode = "'" . $_POST['epName'] . "'" . " - Season " . $_POST['season'] . ", Episode " . $_POST['epNum'];
+    $rating = $_POST['rating'];
+    $image = selectImage($character, $characterPics);
+    sendImageData($db, $image);
+    sendQuotesData($db, $quote, $character, $episode, $rating);
+    echo '<h1>Quote Successfully Added!</h1>';
```

Comment:
It's a shame the redirect wasn't working for you! But better to get working code into master then refactor with that functionality if you have time!